core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateHistogramIT.java

```diff
@@ -1009,20 +1008,13 @@ public void testSingleValueFieldWithExtendedBoundsTimezone() throws Exception {
 
         DateMathParser parser = new DateMathParser(Joda.getStrictStandardDateFormatter());
 
-        final Callable<Long> callable = new Callable<Long>() {
```

Comment:
can we have a fake `now` here that is reproducible or do we need `System.currentTimeMillis()`?